package.json

```diff
@@ -16,22 +16,20 @@
     "url": "https://github.com/badges/shields"
   },
   "dependencies": {
-    "dot": "~1.0.3",
-    "svgo": "~0.5.1",
-    "canvas": "~1.1.2",
-    "phantomjs": "~1.9.2-6",
-    "es6-promise": "~2.1.0",
-    "request": "~2.55.0",
-    "redis": "~1.0.0",
-    "camp": "~15.5.5",
-    "semver": "~4.3.3",
-    "bower": "~1.4.1",
-    "promise": "~7.0.0"
+    "dot": "^1.0.3",
+    "svgo": "^0.6.1",
+    "canvas": "^1.3.5",
+    "phantomjs": "^1.9.2-6",
+    "request": "^2.67.0",
+    "redis": "^2.4.2",
+    "camp": "^15.10.11",
+    "semver": "^5.1.0",
+    "bower": "^1.7.1"
```

Comment:
It's up to you which one you prefer, but `^` should definitely not break anything with update as it keeps the major version unchanged (since they follow semantic versioning).

I also prefer using `^` to `~` because most developers usually don't bother patching earlier versions if a bug is found in a later version.  That is, suppose a bug is introduced in `1.1.0` but is only found in `1.2.0`, I usually don't expect a patch for `1.1.x` to be release to address this bug.

Let me know if you would like me to change it to `~` regardless since it's up to you in the end :)